client/cli/src/config.rs

```diff
@@ -369,6 +369,11 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 		Ok(None)
 	}
 
+	/// Get maximum number of subscriptions per connection.
+	fn rpc_max_subscriptions_per_connection(&self) -> Result<Option<usize>> {
+		Ok(None)
+	}
+
 	/// Get maximum WS output buffer capacity.
```

Comment:
This should be marked as deprecated maybe?

Reply:
It's already but added https://github.com/paritytech/substrate/blob/8392760e00162cd01d95e4534a1996963b3d8e92/client/service/src/lib.rs#L482-#L526